src/preset-themes/003.json

```diff
@@ -2,44 +2,46 @@
   "title": "003",
   "colors": {
     "toolbar": {
-      "h": 200,
-      "s": 29,
-      "l": 32,
-      "a": 17
+      "r": 58,
+      "g": 89,
+      "b": 105,
+      "a": 1
```

Comment:
Should this be 0.17? (trying to spot check some of these against http://hslpicker.com/)